torch/__init__.py

```diff
@@ -267,6 +267,75 @@ def __index__(self):
 
     # Magic methods installed by torch.fx.experimental.sym_node
 
+    def __round__(self, ndigits=None):
+        return self
+
+    def __truediv__(self, other):
```

Comment:
How did I decide when to manually write magic methods on the SymInt/SymFloat class, as opposed to defining a new method on SymNode and using the metaprogramming to insert in the magic method? The defining characteristic is if the magic method could potentially dispatch to two *different* methods on SymNode. E.g., in the example here, we could either end up at `float_truediv` or `int_truediv`. So the dispatching has to happen on the class itself.